mbed_app.json

```diff
@@ -24,7 +24,8 @@
     },
     "target_overrides": {
         "*": {
-            "platform.stdio-convert-newlines": true
+            "platform.stdio-convert-newlines": true,
+            "platform.stdio-baud-rate": 115200
```

Comment:
@LDong-Arm Why do you increase the baud rate to 115200, this was before removed due to CI invokes this mbedhtrun command with the default baud rate

Reply:
It's 115200 in [examples.json](https://github.com/ARMmbed/mbed-os/blob/master/tools/test/examples/examples.json#L512).
Do you mean the CI script doesn't use `--baud-rate` to set it accordingly?

Reply:
@jamesbeyond Is there a chance we can add support for different baud rates in the CI?
Otherwise we would need to use 9600 for everything, though for more verbose examples I'm not confident this rate is always enough.

Reply:
Once we add support to run examples on targets, we should make sure it passes `--baud-rate` with value from examples.json IMO.